solutions/devsprint-firas-safa-1/GitHubApp/Screens/List/ListView.swift

```diff
@@ -8,29 +8,27 @@
 import UIKit
 
 struct ListViewConfiguration {
-
     let listItems: [String]
 }
 
 final class ListView: UIView {
 
     private let listViewCellIdentifier = "ListViewCellIdentifier"
+    private let repositoryCellHeight: CGFloat = 70
```

Comment:
These constants can be static and are immediately related to the `RepositoryCell` so they are better off inside that object / class.